src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -1456,9 +1456,9 @@ public static void Remove_Invalid()
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Remove(-1, 0)); // Start index < 0
             Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(0, -1)); // Length < 0
 
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(6, 0)); // Start index + length > 0
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(5, 1)); // Start index + length > 0
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(4, 2)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(6, 0)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(5, 1)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(4, 2)); // Start index + length > 0
```

Comment:
Looking at the CoreCLR change:
- The coreclr change changed the name of `AddYears` argument name. It needs to reverted. Argument names are part of the contract.
- The name changes need to be ported to CoreRT so that we are consist

I think that the best course of action is to revert the offending part of the change in CoreCLR so that we can deal with it separately from the FileStream changes. I am going to do it.